packages/frontend/src/view/pages/forcedactions/ForcedTradePage.tsx

```diff
@@ -0,0 +1,100 @@
+import React from 'react'
+
+import { ArrowDownIcon } from '../../assets/icons/ArrowIcon'
+import { Button } from '../../components/common/Button'
+import { Card } from '../../components/common/Card'
+import { Link } from '../../components/common/Link'
+import { OrderedList } from '../../components/common/OrderedList'
+import { Page } from '../../components/common/page/Page'
+import { ForcedActionCard } from '../../components/forcedaction/ForcedActionCard'
+import { AmountInput } from '../../components/forcedaction/form/AmountInput'
+import { FormId } from '../../components/forcedaction/form/ids'
+import { PriceInput } from '../../components/forcedaction/form/PriceInput'
+import { TotalInput } from '../../components/forcedaction/form/TotalInput'
+import { reactToHtml } from '../../reactToHtml'
+import {
+  ForcedActionFormProps,
+  serializeForcedActionsFormProps,
+} from './ForcedActionFormProps'
+
+function ForcedTradePage(props: ForcedActionFormProps) {
+  const instructions = [
+    'You create a trade offer using this form. This is fully off-chain and does not require any gas fees.',
+    'The trade offer will now be visible to all users of the system. You should seek out another user to accept the offer and become a counterparty to the trade. Accepting the offer is also fully off-chain and does not require any gas fees.',
+    <span>
+      <span className="text-yellow-300">
+        Once the offer is accepted you can submit a trade request.
+      </span>{' '}
+      This is achieved through a mechanism called forced trades,{' '}
+      <Link href="https://google.com">link to docs</Link>.
```

Comment:
https://docs.starkware.co/starkex/perpetual/perpetual-trading-forced-withdrawal-and-forced-trade.html#forced_trade